indexing-service/src/test/java/org/apache/druid/indexing/firehose/IngestSegmentFirehoseFactoryTimelineTest.java

```diff
@@ -91,9 +91,7 @@
           new JSONParseSpec(
               new TimestampSpec(TIME_COLUMN, "auto", null),
               new DimensionsSpec(
-                  DimensionsSpec.getDefaultSchemas(Arrays.asList(DIMENSIONS)),
```

Comment:
`new DimensionsSpec(DimensionsSpec.getDefaultSchemas(dimensionNames))`
seems like a very frequent construct (atleast in tests).
Should we just a `List<String>` constructor to `DimensionsSpec`?
Or maybe a static `DimensionsSpec.forDimensionNames(dimensionNames)`?

Reply:
I could have done it, but added something similar for the builder instead (`Builder.setDefaultSchemaDimensions()`) to encourage people to use the builder instead. Does this make sense to you?

Reply:
Yes, that makes sense.